lib/scalingo/api/response.rb

```diff
@@ -60,11 +38,17 @@ def paginated?
       end
 
       def operation
+        if operation? && client.respond_to?(:operations)
```

Comment:
I don't really understand how the `client.respond_to?(:operations)` can succeed. When is this `operations` defined?

Reply:
`client` here refers to an instance of `Scalingo::Regional`, which has an `operations` method defined on it (that returns a `Scalingo::Regional::Operations` instance).

async operations are only supported for the regional API, so this is a bit of a safe guard; since the detection is only based on the Location header, this is here to avoid considering auth or billing responses that includes such a header (not the case today, but who knows) as operations